js/event.js

```diff
@@ -560,46 +632,75 @@ export const addEventUPSubmit = () => {
             focus = false;
             hasError = true;
         }*/
-        if(email && /\S+@\S+\.\S+/.test(email) === false) {
-            errorMessage('UPEmail', '<span data-i18n="settingsHelpers.emailFormat">'+translateText('settingsHelpers.emailFormat')+'</span>', focus);
-            if(focus) document.getElementById('UPPhoneNumber11').focus();
-            focus = false;
-            hasError = true;
-        }
-        if(email2 && email2.value && /\S+@\S+\.\S+/.test(email2.value) === false) {
-            errorMessage('UPEmail2', '<span data-i18n="settingsHelpers.emailFormat">'+translateText('settingsHelpers.emailFormat')+'</span>', focus);
-            if(focus) document.getElementById('UPPhoneNumber21').focus();
-            focus = false;
-            hasError = true;
-        }
-        if(email3 && email3.value && /\S+@\S+\.\S+/.test(email3.value) === false) {
-            errorMessage('UPAdditionalEmail2', '<span data-i18n="settingsHelpers.emailFormat">'+translateText('settingsHelpers.emailFormat')+'</span>', focus);
-            if(focus) document.getElementById('UPAdditionalEmail2').focus();
+        const _emailAddressValidation = await emailAddressValidation({
```

Comment:
Only flagging this because I don't think we use the `_` private identifier in other parts of the codebase.